week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -42,20 +42,76 @@ My sources say no.
 Outlook not so good.
 Very doubtful.
 */
-
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+
+function shakeBall() {
+  console.log("The ball has shaken!");
+  return answer[Math.floor(Math.random())];
+};
```

Comment:
In this function, it should return the log and apply the math.random at the possible answers variable that you have created outside of the function.